### DISCUSSION TITLE
Aviso para o curso Fundamentos do Azure, Git, GitHub e DevOps

### DISCUSSION THREAD

**Discussion by mathfrance** [balta-io/forum#277] [Tira Dúvidas] [answered]
Olá @andrebaltieri! Gostaria só de dar um aviso:
- No curso: "Fundamentos do Azure, Git, GitHub e DevOps" Módulo: "Nuget e GitHub Packages" Aula: " Apresentando o GitHub Packages" o vídeo está duplicado com o anterior, a aula mesmo está começando à partir da minutagem 4:47. Apenas avisando caso ainda não estivesse ciente. Abs.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @mathfrance, como vai?

Muito obrigado pelo feedback 💜
